### ISSUE TITLE
mutationWithClientId should support a description argument

### ISSUE THREAD

**Issue by angelf** [graphql/graphql-relay-js#57]
Mutations are just fields, so they can be documented with a 'description' as other fields. 

I think mutationWithClientId should support a 'description' arg and pass it on to the created field.

**Comment by rogchap** [comment]
`mutationWithClientId` returns a simple object so it's easy to add a description for your mutations:

``` javascript
createUser: Object.assign(
  { description: 'Creates a new user' },
  mutationWithClientId({
    // ... 
  })
)
```

What's slightly harder is adding a description to the `Input` and `Payload` objects

**Comment by chentsulin** [comment]
This could be closed by #61

**Comment by wincent** [comment]
Indeed, fixed by #61.
